source/includes/_entitytoentityflow.md

```diff
@@ -0,0 +1,84 @@
+| token        | _string_  | `btc`                                                                                     |                                       |
+| window       | _string_  | `1h` or `1d`                                                                              |
+| from_entity  | _string_  | A miner from the table that we support
+| to_entity    | _string_  | An exchange from the table that we support
+| from_date \* | _string_  | Start date of returned data specified as YYYY-MM-DD (ISO date format)                     |
+| to_date \*   | _string_  | End date of returned data specified as YYYY-MM-DD (ISO date format)                       |
+| limit \*     | _integer_ | The number of entries returned before the latest data point (or the to_date if specified) |
+
+Note: All params with a \* are optional and `limit` is only available in the JSON return format
+
+### Data Overview
+
+| Field                                   | Type      | Description                                                                                                                                                                                                               |
+| --------------------------------------- | --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| avg_txn_value                           | _decimal_ | The average amount BTC transferred per transaction into the given exchange on this date.                                                                                                                                  |
+| avg_txn_value_usd                       | _decimal_ | The USD value of the average amount of BTC transferred per transaction into the given exchange on this date.                                                                                                          |
+| date                                    | _string_  | The date in _YYYY-MM-DD_                                                                                                                                                                                                  |
+| datetime *                              | _string_  | The hour of the day in datetime format YYYY-MM-DD HH:MM:SS (UTC time zone). This is an optional field and appears when window is `1h`                                                                               |
+| hour *                                  | _string_  | The hour of the day in _HH:MM:SS_ (UTC time zone). This is an optional field and appears when window is `1h`                                                                                                        |
+| number_of_txns                          | _integer_ | The number of transactions of sending BTC into an exchange on this date/hour
```

Comment:
remove 'of' in the middle - "The number of transactions sending BTC into an exchange on this date/hour"